specification/flag-evaluation/flag-evaluation.md

```diff
@@ -0,0 +1,195 @@
+# Flag Evaluation API
+
+## Overview
+
+The `evaluation API` allows for the retrieval of feature flag values, independent of any flag control plane or vendor. In the absence of a feature flag system implementation (a [provider](../provider/providers.md)) the `evaluation API` uses the "No-op provider", which simply returns the supplied default flag value.
```

Comment:
minor nit, should we rephrase this as `return of feature flag values` in place of `retrieval`. 

Is it worth calling out optional reason/metadata of why it was returned/error info?

Reply:
Thinking about it more, I think "evaluation" is probably the best word. What do you think?

I also agree it's worth mentioning the metadata, yes.